README.md

```diff
@@ -1,2 +1,26 @@
-# hw01
-Repository for first CFSS homework assignment
+## Some facts about me
+* **Name**: Soo Wan Kim
+* **Born**: Seoul, South Korea
+* **Current residence**: Chicago, IL
+
+## Education/Career
+* **Undergraduate**: University of Chicago Class of 2014, Economics
+* **Current**: [MAPSS](https://mapss.uchicago.edu/), focusing on international relations and computational social science
+* **Research interests**: international security, terrorism
+
+## Timeline
+##### I haven't lived that long.
+* *Summer 2010*, Albany NY - graduated high school
+* *Fall 2010* - started college
+* *Spring 2014* - first RA job
+* *Summer 2014* - graduated college
+* *Fall 2016* - started master's program
+* *Winter 2017* - enrolled in Computing for the Social Sciences
+
+![](https://m.popkey.co/4ab02f/6gMO_f-maxage-0.gif)
+
+## Making this page
+Setting up everything on Git and RStudio was easy since I had used them previously. 
```

Comment:
Did you intend for these sentences to be separated by line breaks? If so, you need to add a blank line between each one, otherwise Markdown merges them all back together. Alternatively, just make it a list like above